include/odp/api/spec/ipsec.h

```diff
@@ -310,6 +310,12 @@ typedef struct odp_ipsec_config_t {
 	/** IPSEC outbound processing configuration */
 	odp_ipsec_outbound_config_t outbound;
 
+	/** Enable stats collection
+	 *
+	 * @see odp_ipsec_stats(), odp_ipsec_stats_multi()
```

Comment:
This could say that the default value is false.

Reply:
Agreed. Will make this change with next version.